dashboard/app/access.go

```diff
@@ -49,6 +49,16 @@ func checkAccessLevel(c context.Context, r *http.Request, level AccessLevel) err
 // AuthDomain is broken in AppEngine tests.
 var isBrokenAuthDomainInTest = false
 
+func emailInAuthDomains(email string, authDomains ...string) bool {
+	for _, authDomain := range authDomains {
+		if authDomain != "" && strings.HasSuffix(email, authDomain) {
```

Comment:
Let's also prepend `@` to `authDomain` if there isn't one.

Reply:
We have a "checkConfig(cfg *GlobalConfig)" function.
It may be a better place for this sanity check.
WDYT?